xla/service/while_loop_unroller.cc

```diff
@@ -737,7 +734,10 @@ absl::StatusOr<bool> WhileLoopUnroller::Run(
   }
   XLA_VLOG_LINES(3, "WhileLoopUnroller::Run(), before:\n" + module->ToString());
   bool changed = false;
-
+  // Allow user provided thresholds.
+  kUnrollTripCountThreshold = unroll_config_.trip_count_threshold;
```

Comment:
ditto

Reply:
Opted to leave as is to not route unroll_config through the pass which would be many changes.